<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetMain</class>
 <widget class="QDialog" name="DialogSetMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>174</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Set main attribute</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>Attribute name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>First column:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="SpinSecond">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Set to -1 to use zero value.</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="SpinFirst">
     <property name="toolTip">
      <string>Set to -1 to use zero value.</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="BtnOk">
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Second column:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="BtnCancel">
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="ComboAttributeName"/>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="CheckUseFirst">
     <property name="text">
      <string>Use first valid</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="CheckUseSecond">
     <property name="text">
      <string>Use second valid</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogSetMain</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnOk</sender>
   <signal>clicked()</signal>
   <receiver>DialogSetMain</receiver>
   <slot>slAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckUseFirst</sender>
   <signal>clicked(bool)</signal>
   <receiver>SpinFirst</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckUseSecond</sender>
   <signal>clicked(bool)</signal>
   <receiver>SpinSecond</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slUseFirstChange()</slot>
  <slot>slUseSecondChange()</slot>
  <slot>slAccept()</slot>
 </slots>
</ui>
